src/fmt.zig

```diff
@@ -517,6 +534,7 @@ test "Syntax grab bag" {
         \\    [1, 2, 3, .. as rest] { 123 }
         \\    3.14 { 314 }
         \\    (1, 2, 3) { 123 }
+        \\    {foo: 1, bar: 2} { 12 }
```

Comment:
Did we agree to put no spaces around pattern records? What's the planned rule for
- record patterns
- record expressions
- record types

Reply:
This isn't a fixed decision, we can move to it whenevs.  I just didn't do it yet

Reply:
I prefer with spaces

Reply:
I just did it